dali/dfuplus/CMakeLists.txt

```diff
@@ -69,5 +69,3 @@ target_link_libraries ( dfuplus
          xmllib
          esphttp
     )
-
```

Comment:
I would completely revert the changes and not include whitespace changes.  The whitespace cleanup is fine, but I wouldn't include it in this PR.